cloudformation/main.yaml

```diff
@@ -267,26 +267,33 @@ Resources:
               DD_HOSTNAME="agentless-scanning-$IMDS_AWS_REGION-$IMDS_INSTANCE_ID"
               DD_SITE="${DatadogSite}"
               DD_API_KEY="$(jq -r '.SecretString' <<< "$AWS_SECRET_JSON")"
-              DD_AGENT_MINOR_VERSION="52.0"
-              DD_SCANNER_VERSION="0.11.0"
+              DD_AGENT_MINOR_VERSION="53.0"
+              DD_AGENTLESS_VERSION="0.11"
```

Comment:
You also need to make this change in `modules/user_data/main.tf` (for the Terraform template).